lib/statsd/instrument/metric_expectation.rb

```diff
@@ -0,0 +1,30 @@
+class StatsD::Instrument::MetricExpectation < StatsD::Instrument::Metric
+
+  attr_accessor :times, :ignore_tags
+
+  # Initializes a new MetricExpectation instance.
```

Comment:
I'd say that's pretty obvious :) we usually don't add boilerplate comments like this